src/app/api/callbacks/ussd/ussd-session-handler.ts

```
@@ -0,0 +1,54 @@
+  static async sessionData(sessionId: string): Promise<USSDSessionData> {
+    const unitId = await this.getUnitId(sessionId);
+    const amount = await this.getAmount(sessionId);
+    const phoneNumber = await this.getPhoneNumber(sessionId);
+
+    return {
+      unitId,
+      amount,
+      phoneNumber,
+    };
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)